src/CrudTrait.php

```diff
@@ -80,7 +80,7 @@ public function addFakes($columns = ['extras'])
                 $column_contents = json_decode($this->{$column});
             }
 
-            if ((is_array($column_contents) || $column_contents instanceof Traversable) && count($column_contents)) {
+            if ((is_array($column_contents) || is_object($column_contents) || $column_contents instanceof Traversable) && count($column_contents)) {
```

Comment:
Actually, @tswonke has a point - that `is_object` fairly much makes the following `instanceof` clause redundant.

We could remove that clause I think.

Reply:
I think the test for `count` also isn't necessary. `count($someObj)` always returns 1 and if array is empty the foreach will not be triggered once...

I wonder if there is any other convenient way to check if a variable can be "foreached"?
(But the test for `is_array` and `is_object` seems completely sufficient)